src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,45 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.ArrayList;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int START_SIZE = 0;
+    private K key;
+    private V value;
+    private ArrayList<K> keys;
+    private ArrayList<V> values;
```

Comment:
Don’t use any kind of List, Set or Map
```suggestion
    private K [] keys;
    private V [] values;
```